client/src/utils/fight/skillActivate.ts

```diff
@@ -37,6 +37,9 @@ const skillActivate = async (
   if ([SkillId.cryOfTheDamned, SkillId.fierceBrute, SkillId.hammer].includes(step.s)) {
     const animationEnded = brute.animation.waitForEvent('strengthen:end');
 
+    // Note if brute was stunned
+    const bruteWasStunned = brute.animation.animation === 'death';
```

Comment:
This could fail if the brute was recovering from another animation, on its way to go back to the death animation.
Instead, it would be better to store the stun status in the AnimationFighter object, just like the trapped status, and read that.